.vscode/settings.json

```diff
@@ -5,7 +5,7 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "eslint.workingDirectories": [{ "mode": "auto" }],
   "javascript.format.enable": false,
-  "window.zoomLevel": 1,
+  "window.zoomLevel": 0.5,
```

Comment:
so you need to commit the .vscode/settings.json, before you commit this file, to not have these changes applying yo all of us